models/edr/dbt_artifacts/dbt_exposures.sql

```diff
@@ -1,8 +1,8 @@
 {{
   config(
-    materialized='table',
+    materialized='incremental',
     transient=False,
-    post_hook='{{ elementary.upload_dbt_exposures() }}'
+    post_hook=after_commit('{{ elementary.upload_dbt_exposures() }}')
```

Comment:
`after_commit` is needed because I'm using transactions (affects only Redshift, as it's the only adapter that actually uses transactions)